pkg/pillar/base/execwrapper.go

```diff
@@ -0,0 +1,209 @@
+package base
```

Comment:
Please add a the Copyright+License two lines.

Reply:
Done.